docs/pages/guides/pages/built-in-pages.mdx

```diff
@@ -3,22 +3,66 @@ import { Screenshot } from "@/components/Screenshot"
 
 # Built-in Pages
 
-Auth.js comes by default with a set of pages that are presented to the user as they go through their authentication journey (sign up, sign in, sign out, error, etc...). This is helpful so that you don't need to write those from scratch when using the library first time.
+Auth.js comes by default with a set of pages that are presented to the user as they go through their authentication journey (sign up, sign in, sign out, error, etc...). This is helpful so that you don't need to write those from scratch when using the library first time. The UI created is based on the providers specified in your configuration file.
+
+```ts filename="lib/providers.ts"
+import Github from "next-auth/providers/github"
+import Credentials from "next-auth/providers/credentials"
+import type { Provider } from "next-auth/providers"
+
+export const providers: Provider[] = [
+  Github,
+  Credentials({
+    credentials: {
+      username: { label: "Username" },
+      password: { label: "Password", type: "password" }
+    },
+    async authorize(credentials) {
+      // mplement some logic to check if the user exists.
+      return credentials
+    }
+  })
+]
+```
 
 <Code>
```

Comment:
I think you're right, the `SignInButton` example wasn't very helpful here. But I dont think these "how to set providers" examples are very useful here either.

I think we could probably just remove all of these examples for better clarity on this page :ok_hand:

Reply:
I hadn’t read this comment before, but I think you’re right. I will wait for your response to my previous comment and make the necessary changes.